Common_Files/Domain.ipynb

```diff
@@ -6,96 +6,37 @@
     "\n",
-    "<img src=\"./Images/Fig2.jpg\" width=\"500\">\n",
-    "                                            \n",
-    "                                               Fig.2: The regions chosen to refine the mesh.\n",
+    "The rule of thumb is to have a high mesh resolution close to the rectangle area where the flow field behavior is complex due to formation of boundary layer. These areas are highlighted in Fig. 2. Then mesh resolution is gradually decreased toward the outer regions where the flow field physics and behavior are rather simple and capturing them numerically would be straight forward. For even more complex flow field such as turbulent boundary layer there are specific techniques and guidelines to set the mesh resolution around the rectangle's wall, which will be introduced in other case studies as needed. Using the current approach one created a well-defined CFD domain geometry with high mesh resolution close to rectangle's boundary, where the physics of the flow are more complex (i.e. boundary layer region) and gradually decreased it to bounds of the domain, where flow field is uniform and simple.\n",
     "\n",
+    "<img src=\"./Images/Fig2.jpg\" width=\"500\">\n",
+    "Fig.2: The regions chosen to refine the mesh.\n",
     "\n",
-    "The element size chosen to refine the mesh was 25mm, which will give 80 elements along the length of the rectangle and 40 elements along the width of the rectangle. This type of distribution is not ideal since a smaller element size should be assigned so that there are more than 100 elements along the length of the rectangle. However, the results were close to the experiment data conducted by a few CFD groups, thus it is acceptable due to limitation in computer resources. It is worth noting that the two slender rectangles on the left should have been coarsely meshed, because the inlet flow is uniform, thus more calculation in those two inlet region is redundant.\n",
-    "Fig.3 shows the fully meshed body and Fig.4 shows the zoom in view of the fine mesh regions.\n",
+    "The element size chosen to refine the mesh was $25 [mm]$, which leads to 80 elements along the length and 40 elements along the width of the rectangle. Fig. 4 shows the zoom in view of the meshed region aroud the central rectangle. It is worth noting that the two slender rectangles on the left should have been coarsely meshed, because the inlet flow is uniform, thus more calculation in those two inlet region is redundant. Fig.3 shows the fully meshed CFD domain.\n",
```

Comment:
Add detail of mesh size/distribution on for the subdomains.
Add description about mesh element type and why you chose them.